### PR TITLE
Deploying ArgoCD on MiniKube with Terraform and Helm

### PR DESCRIPTION
**Pull request by donia1337**

### CODE REVIEW

File: minikube-projects/terraform/.terraform.lock.hcl
Comment (Aimdrow) [inline minikube-projects/terraform/.terraform.lock.hcl:1]: Remove

File: minikube-projects/terraform/.terraform/providers/registry.terraform.io/hashicorp/helm/2.13.2/darwin_arm64/LICENSE.txt
Comment (Aimdrow) [inline minikube-projects/terraform/.terraform/providers/registry.terraform.io/hashicorp/helm/2.13.2/darwin_arm64/LICENSE.txt:1]: Remove

File: minikube-projects/terraform/.terraform/providers/registry.terraform.io/hashicorp/helm/2.13.2/darwin_arm64/terraform-provider-helm_v2.13.2_x5
Comment (Aimdrow) [inline minikube-projects/terraform/.terraform/providers/registry.terraform.io/hashicorp/helm/2.13.2/darwin_arm64/terraform-provider-helm_v2.13.2_x5:1]: Remove

File: minikube-projects/terraform/terraform.tfstate
Comment (Aimdrow) [inline minikube-projects/terraform/terraform.tfstate:1]: Remove

**Review by Aimdrow — CHANGES_REQUESTED** [review]
We do not need the statefiles, as the state will be different next time you boot up minikube

**Comment by donia1337** [comment]
> We do not need the statefiles, as the state will be different next time you boot up minikube

The statefiles have been removed in this case - as minikube is temporary.

A reference it .gitignore has been added, so they will not be pushed again. Yes, I know to be aware, if I create a future resource, that I want to keep track of, I will need to either a) remove this reference or b) force push the state files.

**Review by Aimdrow — APPROVED** [review]
LGTM

**Pull request merged**